src/porepy/viz/plot_grid.py

```diff
@@ -122,33 +116,35 @@ def save_img(
 
 def plot_sd(
     sd: pp.Grid,
-    cell_value: Optional[np.ndarray],
-    vector_value: Optional[np.ndarray],
-    info: Optional[str],
+    cell_value: Optional[np.ndarray] = None,
+    vector_value: Optional[np.ndarray] = None,
+    info: Optional[str] = None,
     **kwargs,
 ) -> None:
-    """
-    Plot data on a subdomain and provided data.
-
-    Args:
-        sd (pp.Grid): Subdomain
-        cell_value (np.ndarray): cell-wise scalar values, will be represented by the color
-        of the cells.
-        vector_value (np.ndarray): vector values, one 3d vector for each cell or for each
-            face (see the _quiver function).
-        info (str, optional): Which geometry information to display, see add_info.
-        kwargs (optional): Keyword arguments:
+    """Plot data on a subdomain and provided data.
+
+    Parameters:
+        sd: Subdomain
+        cell_value: cell-wise scalar values, will be represented by the color of the
+            cells.
+        vector_value: vector values, one 3d vector for each cell or for each face (see
+            the _quiver function).
+        info: Which geometry information to display, see add_info.
+        kwargs: Keyword arguments:
             fig_size: Size of figure.
             fig_num: The number of the figure.
-            color_map: Limits of the cell value color axis.
+            fracturewidth_1d: Width of fractures in 1d.
+            color_map: Color map as str or matplotlib color map. Defaults to jet.
+            color_map_limits: Limits of the cell value color axis.
```

Comment:
```suggestion
            color_map_limits: Limits of the cell value color axis, given as a numpy array
                with min and max values.
```suggestion
            color_map_limits: Limits of the cell value color axis.
```
```